tests/test_app/tests/managementcommand_tests.py

```diff
@@ -86,7 +86,7 @@ def ls_test(*args, expected):
             lines = out.read().splitlines()
             self.assertEqual(sorted(lines), sorted(expected))
 
-        test_data = (
+        test_data: list[tuple[list[str], list[str]]] = [
```

Comment:
This isn't strictly necessary for the Pyright tests, but it's a more sensible data structure and made MyPy happier.